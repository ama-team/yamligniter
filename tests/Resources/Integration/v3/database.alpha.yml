active_group: main
db:
  main:
    hostname: mysql
    username: application
    password: password
    database: application_things
    char_set: utf8mb4
    dbcollat: utf8mb4_general_ci
    failover:
      - hostname: failover-1
        username: failover
        password: fuxxored
        database: application_things
        char_set: utf8mb4
        dbcollat: utf8mb4_general_ci
      - hostname: failover-2
        username: failover
        password: fuxxored
        database: application_things
        char_set: utf8mb4
        dbcollat: utf8mb4_general_ci

