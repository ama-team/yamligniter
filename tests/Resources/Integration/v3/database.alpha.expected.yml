query_builder: true
active_group: main
db:
  main:
    dsn: ''
    hostname: mysql
    username: application
    password: password
    database: application_things
    dbdriver: mysqli
    dbprefix: ''
    pconnect: true
    db_debug: true
    cache_on: false
    cachedir: ''
    char_set: utf8mb4
    dbcollat: utf8mb4_general_ci
    swap_pre: ''
    encrypt: false
    compress: false
    stricton: false
    failover:
      - dsn: ''
        hostname: failover-1
        username: failover
        password: fuxxored
        database: application_things
        dbdriver: mysqli
        dbprefix: ''
        pconnect: true
        db_debug: true
        cache_on: false
        cachedir: ''
        char_set: utf8mb4
        dbcollat: utf8mb4_general_ci
        swap_pre: ''
        encrypt: false
        compress: false
        stricton: false
        # it's easier to keep than to bear with this little inaccuracy
        failover: []
      - dsn: ''
        hostname: failover-2
        username: failover
        password: fuxxored
        database: application_things
        dbdriver: mysqli
        dbprefix: ''
        pconnect: true
        db_debug: true
        cache_on: false
        cachedir: ''
        char_set: utf8mb4
        dbcollat: utf8mb4_general_ci
        swap_pre: ''
        encrypt: false
        compress: false
        stricton: false
        failover: []

